tutor/src/screens/teacher-dashboard/month.jsx

```diff
@@ -147,7 +147,8 @@ class CourseMonth extends React.Component {
     const daysOfDuration = calendarDuration.iterate('days');
 
     while (daysOfDuration.hasNext()) {
-      mods.push(makeModForDate(daysOfDuration.next(), this.props.date, this.state));
+      // mods.push(makeModForDate(daysOfDuration.next(), this.props.date, this.state));
+      mods.push(makeModForDate(daysOfDuration.next(), this.props.date));
```

Comment:
Not sure about this change, was going to fix it later, taking out state since it's not defined in this component